model/src/test/kotlin/utils/FindingsMatcherTest.kt

```diff
@@ -37,8 +37,10 @@ import kotlin.random.Random
 
 private fun Collection<LicenseFindings>.getFindings(license: String) = single { it.license == license }
 
+private const val ROOT_LICENSE_FILE = "a/LICENSE"
```

Comment:
Commit message typo in "Exctract".

Also, I wonder if `ROOT_LICENSE_FILE` is a good name here. What makes "a/LICENSE" the root license instead of "b/LICENSE" below?

Reply:
I'm open for suggestions.

Reply:
How about eliminating all occurences of "b/LICENSE" and then also changing the value of ROOT_LICENSE_FILE to just "LICENSE"? Would that solve your issue?

Reply:
Actually, my preferred solution would be to not introduce a variable here at all, and leave the code as-is. I see no benefit of introducing a variable here, esp. if not doing the same for all other strings that occur multiple times in the file, like the named "b/LICENSE" and also "some/other/file".

Reply:
Have adjusted the commit, does it help ?

Reply:
I like the variables actually because the create a link between the LicenseFileMatcher created at the top of the class and the test cases using it. So if your objection isn't blocking I'd like to stick with the variables ?!